test/functional/README.md

```diff
@@ -63,10 +63,11 @@ don't have test cases for.
 - Avoid stop-starting the nodes multiple times during the test if possible. A
   stop-start takes several seconds, so doing it several times blows up the
   runtime of the test.
-- Set the `self.setup_clean_chain` variable in `set_test_params()` to control whether
-  or not to use the cached data directories. The cached data directories
-  contain a 200-block pre-mined blockchain and wallets for four nodes. Each node
-  has 25 mature blocks (25x50=1250 BTC) in its wallet.
+- Set the `self.setup_clean_chain` variable in `set_test_params()` to `True` if
+  you don't want to use the cached data directories (defaults to `False`). The
+  cached data directories contain a 200-block pre-mined blockchain and wallets
+  for four nodes. Each node has 25 mature blocks (25x50=1250 BTC) in its wallet.
```

Comment:
Strictly speaking, it doesn't contain any `wallets`, only `'chainstate', 'blocks'`

Reply:
possible alternative for the first sentence:

"...to `True` to initialize an empty blockchain, rather than load a premined blockchain from cache with the default value of `False`."

Reply:
And wallets also doesn't have blocks in them :) I improved the clunky parts of that paragraph a bit further, taking @jonatack 's suggested sentence with a minor tweak.